scopes/lanes/ui/overview/lane-id.module.scss

```diff
@@ -0,0 +1,22 @@
+.lane {
+  display: flex;
+  align-items: center;
+  min-width: 0;
+  padding: 24px 0px 24px 32px;
+  font-size: var(--bit-p-s);
+  .laneId {
+    overflow: hidden;
+    white-space: nowrap;
+    padding: 0px 8px;
+    font-size: var(--bit-p-xs);
```

Comment:
use the [Ellipsis](https://bit.dev/teambit/design/ui/styles/ellipsis/~code/index.ts) class